src/main/java/com/carleton/CapstoneSystem/DTO/DayDTO.java

```diff
@@ -0,0 +1,61 @@
+package com.carleton.CapstoneSystem.DTO;
+
+import com.carleton.CapstoneSystem.models.Month;
+import com.carleton.CapstoneSystem.models.Time;
+import com.carleton.CapstoneSystem.models.WeekDay;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+
+public class DayDTO {
+    @JsonProperty
+    private WeekDay weekDay;
+    @JsonProperty
+    private Month month;
```

Comment:
We can just use the Date class instead of having month, dayDate. It would be very hard to send the information like this in the client side.

Reply:
Actually it’s fine this way